packages/mobile/src/account/InviteReview.test.tsx

```diff
@@ -1,6 +1,6 @@
 jest.useFakeTimers()
 
-import Button from '@celo/react-components/components/Button'
+import mockButton from '@celo/react-components/components/Button'
```

Comment:
this is that scoping jest thing - you can apparently get around it by just calling stuff "mock" whatever

Reply:
gotcha gotcha - so this looks okay to you?